### ISSUE TITLE
LakhMIDIDataset: Downloaded file has a different size from the expected one

### ISSUE THREAD

**Issue by cifkao** [salu133445/muspy#57]
I downloaded the Lakh MIDI dataset with:
```python
muspy.datasets.LakhMIDIDataset(LMD_FULL_DIR, download_and_extract=True)
```
I got the following error:
```pytb
muspy/datasets/utils.py in download_url(url, path, overwrite, size, md5, sha256, verbose)
    179     if size is not None and path.stat().st_size != size:
    180         raise RuntimeError(
--> 181             "Downloaded file has a different size from the expected one."
    182         )
    183     if md5 is not None and not check_md5(path, md5):

RuntimeError: Downloaded file has a different size from the expected one.

ipdb> p path.stat().st_size
1768163879
ipdb> p size
35727800
```

The checksum matches, though.

**Comment by salu133445** [comment]
Thanks for reporting! This has been fixed by 497fb29.

**Comment by salu133445** [comment]
Maybe we should warn the users rather than raising an error.
